src/web/mjs/connectors/CosmicScansID.mjs

```diff
@@ -0,0 +1,13 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class CosmicScans extends WordPressMangastream {
```

Comment:
* Class must match filename
* This class is already used by english cosmic scans btw